pages/platform/kubernetes-k8s/customizing-kubeproxy/guide.en-gb.md

```diff
@@ -0,0 +1,243 @@
+   }
+}
+```
+
+> [!primary]
+>
+> This API call generate a `configMap` that will be used by the kube-proxy component.
+>
+> To access this `configMap` you can execute the `kubectl get cm kube-proxy -n kube-system -o yaml` command.
+>
+> The kube-proxy `configMap` in new Kubernetes clusters includes `config.conf` entry.
+
+Both IPVS and iptables specific configuration can be set at the same time and kube-proxy will select the one to use according to the mode value.
+
+If a field is not specified in the API payload, it will not be present in the config file and let kube-proxy use its default value (`kubeProxyMode` default is 'iptables').
+
+You can take a look to the [Kube-proxy default values](https://github.com/kubernetes/kubernetes/blob/master/pkg/proxy/apis/config/v1alpha1/defaults.go#L38) for more information.
+
+- Reset a Kubernetes cluster (all Kubernetes data will be erased (pods, services, configuration, etc), nodes will be either deleted or reinstalled)
+
+> [!primary]
+>
+> `kubeProxyMode` cannot be modified, you need to reset your Kubernetes cluster.
+>
+
+> [!api]
+>
+> @api {POST} /cloud/project/{serviceName}/kube/{kubeID}/reset
+> 
+
+```json
+{
+    "region": "GRA5",
+    "name": "my-super-cluster",
+    "kubeProxyMode": "ipvs",
+    "customization": {
+      "kubeProxy":{
+         "iptables":{
+            "minSyncPeriod":"PT1S",
+            "syncPeriod":"PT30S"
+         },
+         "ipvs":{
+            "minSyncPeriod":"PT0S",
+            "scheduler":"rr",
+            "syncPeriod":"PT30S",
+            "tcpFinTimeout":"PT0S",
+            "tcpTimeout":"PT0S",
+            "udpTimeout":"PT0S"
+         }
+      }
+   }
+}
+```
+
+Both kubeProxyMode and customization.* can be modified on cluster reset with the same payload used for creation.
+
+If these fields are not specified, it will reset to default value (ipvs for kubeProxyMode and empty customization).
+
+- Update only `kubeProxy` and keeping existing `apiServer` customization if any:
+
+> [!primary]
+>
+> `kubeProxyMode` cannot be modified by updating an existing cluster, it can only be set on cluster creation and reset.
+
+> [!api]
+>
+> @api {PUT} /cloud/project/{serviceName}/kube/{kubeID}/customization
+> 
+
+```json
+{
+	"kubeProxy": {
+		"iptables": {
+			"minSyncPeriod": "PT60S"
+			"syncPeriod": "PT60S"
+		}
+	}
+}
+```
+
+- Update both apiServer and override kubeProxy configuration:
+
+> [!api]
+>
+> @api {PUT} /cloud/project/{serviceName}/kube/{kubeID}/customization
+> 
+
+```json
+{
+	"apiServer": {
+		"admissionPlugins": {
+			"enabled": [],
+			"disabled": [
+				"AlwaysPullImages"
+			]
+		}
+	},
+	// kubeProxy customization will be OVERRIDDEN (minSyncPeriod will be removed in this example)
+	"kubeProxy": {
+		"iptables": {
+			"syncPeriod": "PT120S"
+		}
+	}
+}
+```
+
+> [!primary]
+>
+> Updating customization.kubeProxy will trigger following actions:
+>
+> - apply updated configMap
+> - then rollout restart kube-proxy so it uses the new configuration
```

Comment:
Should it be:

- rollout/restart
- rollout
- restart
?

Reply:
"kubectl rollout restart" is a command :)